woocommerce/woocommerce-ios
# SPE: Always display category on product details

Closes: https://github.com/woocommerce/woocommerce-ios/issues/8839

## Description

This PR adds "Category" row on product details form even when no categories are selected.

## Testing

1. Build and run the app in debug/alpha mode.
2. Open product details for a product without any categories assigned (or start product creation flow).
3. Confirm "Category" on top level is displayed.
4. Tap "Category" and try updating it.

## Screenshots

before|after
--|--
![Simulator Screen Shot - 1](https://user-images.githubusercontent.com/3132438/220064227-97752f1f-f6fa-48cf-957a-dd86249d22a0.png)|![Simulator Screen Shot - 2](https://user-images.githubusercontent.com/3132438/220064293-ed31b788-9ba5-4190-bcfe-6a0154d41019.png)


---
